File: back/DevC/src/main/java/com/back/devc/domain/auth/dto/signup/SignUpResponse.java
package com.back.devc.domain.auth.dto.signup;

import com.back.devc.domain.member.member.entity.MemberRole;
import com.back.devc.domain.member.member.entity.MemberStatus;

public record SignUpResponse(
        Long userId,
        String email,
        String nickname,
        MemberRole role,
        MemberStatus status
) {
}

File: back/DevC/src/main/kotlin/com/back/devc/domain/auth/dto/signup/SignUpResponse.kt
package com.back.devc.domain.auth.dto.signup

import com.back.devc.domain.member.member.entity.MemberRole
import com.back.devc.domain.member.member.entity.MemberStatus

data class SignUpResponse(
    val userId: Long,
    val email: String,
    val nickname: String,
    val role: MemberRole,
    val status: MemberStatus
)
